Sources/CodeEditor/UXCodeTextViewRepresentable.swift

```diff
@@ -94,6 +94,12 @@ struct UXCodeTextViewRepresentable : UXViewRepresentable {
       parent.source.wrappedValue = textView.string
     }
     
+    #if os(iOS)
```

Comment:
That should probably be an `#if os(macOS) #else ` then?

Reply:
`textViewDidChange(_ textView: UITextView)` is only available on iOS. Not sure about macOS equivalent, but this should work. Personally, I would leave the `textDidChange(_ notification: Notification)` unconstrained, to make sure it works on other/older OS.